config/schema/document_types/asylum_support_decision.json

```diff
@@ -0,0 +1,190 @@
+{
+  "fields": [
+    "tribunal_decision_decision_date",
+    "tribunal_decision_judges",
+    "tribunal_decision_category",
+    "tribunal_decision_sub_category",
+    "tribunal_decision_landmark",
+    "tribunal_decision_reference_number"
+  ],
+  "allowed_values": {
+    "tribunal_decision_judges": [
```

Comment:
I'm deeply concerned about adding allowed-values for a set of judge names here - I worry about the administrative burden whenever a judge gains a title, or changes their name.

This data really belongs in a database, not in git.  I'd like to discuss ways in which we can move towards that before this PR is merged.  Even if the result of the discussion is that we merge the PR for now, I want to have a plan in place for how this is to be improved in future.  (Such discussion is best done in person, I think - I'll be in the office from tomorrow.)